packages/frontend/src/App.tsx

```diff
@@ -93,74 +84,114 @@ const App = () => (
+                                                <CreateProjectSettings />
                                             </Page>
                                         </Route>
-                                        <Route path="/projects/:projectUuid/dashboards/:dashboardUuid">
-                                            <AppBar />
-                                            <Page name={PageName.DASHBOARD}>
-                                                <Dashboard />
-                                            </Page>
-                                        </Route>
-                                        <Route path="/projects/:projectUuid/dashboards">
-                                            <AppBar />
-                                            <Page
-                                                name={PageName.SAVED_DASHBOARDS}
-                                            >
-                                                <SavedDashboards />
-                                            </Page>
-                                        </Route>
-                                        <Route path="/projects/:projectUuid/sqlRunner">
-                                            <AppBar />
-                                            <Page name={PageName.SQL_RUNNER}>
-                                                <SqlRunner />
-                                            </Page>
-                                        </Route>
-                                        <Route path="/projects/:projectUuid/tables/:tableId">
-                                            <AppBar />
-                                            <Page name={PageName.EXPLORER}>
-                                                <ExplorerProvider>
-                                                    <Explorer />
-                                                </ExplorerProvider>
-                                            </Page>
-                                        </Route>
-                                        <Route path="/projects/:projectUuid/tables">
-                                            <AppBar />
-                                            <Page
-                                                name={PageName.EXPLORE_TABLES}
-                                            >
-                                                <ExplorerProvider>
-                                                    <Explorer />
-                                                </ExplorerProvider>
-                                            </Page>
-                                        </Route>
-                                        <Route path="/projects" exact>
-                                            <Projects />
-                                        </Route>
-                                        <Redirect to="/projects" />
+                                        <AppRoute path="/">
```

Comment:
@owlas now we have private routes ( needs to be authenticated ) and app routes ( needs to have completed onboarding ) 
Indentation caused most of the file to change.

Reply:
Nice idea! Really clear